tensorboard/plugins/image/images_plugin.py

```diff
@@ -74,7 +74,7 @@ def get_plugin_apps(self):
     }
 
   def is_active(self):
-    """The images plugin is active iff any run has at least one relevant tag."""
```

Comment:
Am a little unclear if the author really intended this but, to me, it reads properly. Though it is used in very particular way in other parts of computer science (https://en.wikipedia.org/wiki/If_and_only_if), it is sometimes used as stressed `if`

Reply:
I'm pretty sure it's an intentional reference to if and only if.  It is kind of jargon-y though and I don't know that it's really necessary here.  Could also rephrase as e.g. "Returns true if at least one run has a relevant tag, otherwise false."